src/base-ubuntu/.devcontainer/Dockerfile

```
@@ -3,6 +3,17 @@ FROM buildpack-deps:${VARIANT}-curl
 
 LABEL dev.containers.features="common"
 
+ARG VARIANT
+ENV VARIANT=${VARIANT}
```

```suggestion
ARG VARIANT
```

Do we need `VARIANT` at runtime? I wonder if having it at build time should be sufficient in this case?

---

yes that's right, works without env variable declaration as well..